Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -649,6 +649,19 @@ private void SyncThumbs()
             Canvas.SetLeft(_minThumb, relativeLeft);
             Canvas.SetLeft(_maxThumb, relativeRight);
 
+            if (fromMinKeyDown || fromMaxKeyDown)
+            {
+                DragThumb(
+                    fromMinKeyDown ? _minThumb : _maxThumb,
```

Comment:
To clarify, move each of these cases into a separate call back into the KeyDown calls rather than passing in flags to this method.

Reply:
I understand what you mean and I think it can be done without issues.

The reason why I added this to `SyncThumbs` is because I need both `relativeLeft` and `relativeRight` (see [here](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/c9d701b5ef7c2b0b3f812d26849b4c7e690adbb3/Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs#L646)) so, instead of duplicating code, I edited that method.

If you're ok with some little duplication I'll move everything in another method.

Reply:
Yeah, I noticed that afterwards too.  We can leave it all for now, it's all internal anyway.  A comment or two to explain ordering could be handy, but we can investigate later if there's an easier way to order things.  For now, we'll get the fix in the for the release, thanks for fixing this! 👍